test/src/e2e.dynval/setup.ts

```diff
@@ -326,7 +326,7 @@ export const defaultParams = {
     minSetShardOwnersCost: 10,
     minSetShardUsersCost: 10,
     minWrapCccCost: 10,
-    minCustomCost: 10,
+    minCustomCost: 0,
```

Comment:
Would you use `overrideParams` in `withNodes`?

Reply:
Why? I thought making this cost to `0` would make our test closer to the real-world environment.